config/logstash.yml

```diff
@@ -131,6 +131,16 @@
 #
 # queue.checkpoint.interval: 1000
 #
+# ------------ Dead-Letter Queue Settings --------------
+# Flag to turn on dead-letter queue.
+#
+# dlq.enable: false
+#
+# If using dlq.enable: true, the directory path where the data files will be stored.
+# Default is path.data/dlq
+#
+# path.dlq: false
```

Comment:
I know it's WIP, but just an early feedback -- let's expand this as `path.dead_letter_queue` instead of `path.dlq`